### PR TITLE
feat: polish in the footer + tracking README usage section

### PR DESCRIPTION
**Pull request by dylanfareed**

### CODE REVIEW

File: lib/components/Timeline/SecondaryStep.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const SecondaryStep = ({ config }: { config: TrackingConfig }) => {
       >
         <path
           d="M8 4C8 6.20914 6.20914 8 4 8C1.79086 8 0 6.20914 0 4C0 1.79086 1.79086 0 4 0C6.20914 0 8 1.79086 8 4Z"
-          fill="#D7D6D0"
+          fill={config.style.color.iconSecondary}
```
Comment (dylanfareed) [inline lib/components/Timeline/SecondaryStep.tsx:15]: Small fix here

File: lib/components/DrawerFooter/index.tsx [hunk 2]
```diff
@@ -1,51 +1,53 @@
 export const DrawerFooter = () => {
   return (
     <div class="artajs__tracking__footer__wrapper">
-      <div class="artajs__tracking__footer__content">
-        <div class="artajs__tracking__footer__powered__by__text">
-          Powered by
-        </div>
-        <div class="artajs__tracking__footer__powered__by__logo">
+      <a href="https://arta.io/">
```
Comment (dylanfareed) [inline lib/components/DrawerFooter/index.tsx:4]: Updating the "Powered by" sections in both Estimates and Tracking to be more consistent

File: README.md [hunk 3]
```diff
@@ -29,41 +30,107 @@ And you can then import `Arta` object for example:
...
+      duration: 250,
+      easing: "ease-in-out",
+    },
+  },
+  style: {
+    color: {
+        iconPrimary: "blue"
+      }
+    }
+  }
```
Comment (otaviojacobi) [inline README.md:114]: Can we run a quick linter over these so `{}` match properly and we add `;` at the end?

e.g.
```suggestion
const config = {
  animation: {
    in: {
      type: "slide",
      duration: 500,
      easing: "ease-in-out",
    },
    out: {
      type: "slide",
      duration: 250,
      easing: "ease-in-out",
    },
  },
  style: {
    color: {
      iconPrimary: "blue"
    }
  }
};
```

File: README.md [hunk 4]
```diff
@@ -29,41 +30,107 @@ And you can then import `Arta` object for example:
...
+    {
+      depth: 2,
+      width: 36,
+      height: 24,
+      subtype: "painting_unframed",
+      unit_of_measurement: "in",
+      value_currency: "USD",
+      value: 500.00
+    }
+  ]
```
Comment (otaviojacobi) [inline README.md:60]: Same comment regarding linting below

Comment (dylanfareed) [reply README.md]: 👍 updated

image

**Review by otaviojacobi — APPROVED** [review]

**Pull request merged**